less/home-page.less

```diff
@@ -46,24 +95,48 @@
     display: flex;
     justify-content: center;
 
+    @media @phone {
+      display: flex;
+      flex-direction: column-reverse;
+      padding: 20px;
+    }
+
     .walk-text {
       width: 50%;
 
+      @media @phone {
+        width: 100%;
+      }
+
       h3 {
         margin-bottom: 10px;
+
+        @media @phone {
+          margin-top: 20px;
+        }
       }
 
       p {
         margin-bottom: 26px;
       }
 
       .btn {
+        .button(50px, 100px);
+
+        @media @phone {
+          .button(70px, 100%);
+        }
       }
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
+
+      @media @phone {
+        width: 100%;
+        margin: 9 auto;
```

Comment:
missing unit of measurement of `9`

Reply:
margin-left: 4% also needs to be overwritten to center the margin. I would just say margin: 0 for the phone query.